File: server/src/main/java/com/ivantrykosh/app/budgettracker/server/dtos/AccountUsersDto.java
package com.ivantrykosh.app.budgettracker.server.dtos;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * Account users dto
 */
@NoArgsConstructor
@Getter
@Setter
@ToString
public class AccountUsersDto {
    private Long accountUsersId; // Account users ID

    private Long accountId; // ID of account to which users belong

    private String email2; // Second user email

    private String email3; // Third user email

    private String email4; // Fourth user email
}

File: client/app/src/main/java/com/ivantrykosh/app/budgettracker/client/data/remote/dto/AccountUsersDto.kt
package com.ivantrykosh.app.budgettracker.client.data.remote.dto

/**
 * Data class for account users
 */
data class AccountUsersDto(
    val accountUsersId: Long,
    val accountId: Long,
    val email2: String,
    val email3: String,
    val email4: String
)
